joplin/css/layouts/headers.scss

```diff
@@ -48,6 +48,26 @@
   height: 40px !important;
 }
 
+.nav-search li {
+  margin: 0px;
+  padding: 0px;
+  list-style-type: none;
+}
+
+.icon-search:before {
+  margin-top: 5px !important;
+  color: #777 !important;
+}
+
+.icon-search input {
+  color: #262626 !important;
+  background-color: hsla(0,0%,39%,.15) !important;
+}
+
+.icon-search input::placeholder {
+  color: #aaa !important;
+}
```

Comment:
these styles are a rough pass at making the input state static for "inside" the text box or not. We can tackle how we handle /design soon!